app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+  end
+
+  def create
+    @payment = Payment.new(payment_params)
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+
+    if @payment.valid? && @payment.purchase(current_user, @conference, price_in_cents) && @payment.save
```

Comment:
Also, checking the code for `TicketPurchase.purchase`, it seems to always return a truthy value, doesn't it?
Does it belong in the conditional then, or do you want to somehow refactor that method?